src/main/java/com/salesforce/storm/spout/dynamic/VirtualSpout.java

```diff
@@ -271,6 +271,8 @@ public void close() {
 
         startingState = null;
         endingState = null;
+
+        isOpened = false;
```

Comment:
Is this change intentional?  Do we want to allow re-opening a closed instance?  Do we / can we have any validation that a VSpout and its internal consumer is in a sane state to be re-opened?

Reply:
It was intentional, but not for a functional need.  Seemed like something we should do. I always thing of open/close as constructor/destructor's and they should be resumable, but maybe I've been thinking of that wrong the whole time.

Reply:
Yea I don't think that's a safe assumption IMO.